docs/tutorials/regression.ipynb

```diff
@@ -2,204 +2,391 @@
+      ],
+      "text/plain": [
+       "+-----+------+-------+-----+---+----------------+---------------+------------------+--------+\n",
+       "|  id | year | month | day | … | year_renovated | sqft_lot_15nn | sqft_living_15nn |  price |\n",
+       "| --- |  --- |   --- | --- |   |            --- |           --- |              --- |    --- |\n",
+       "| i64 |  i64 |   i64 | i64 |   |            i64 |           i64 |              i64 |    i64 |\n",
+       "+===========================================================================================+\n",
+       "|   0 | 2014 |     5 |   2 | … |              0 |          9176 |             2310 | 285000 |\n",
+       "|   1 | 2014 |     5 |   2 | … |              0 |          8595 |             1750 | 285000 |\n",
+       "|   2 | 2014 |     5 |   2 | … |              0 |          9000 |             1850 | 440000 |\n",
+       "|   3 | 2014 |     5 |   2 | … |              0 |          8942 |             1260 | 435000 |\n",
+       "|   4 | 2014 |     5 |   2 | … |              0 |         10836 |             1450 | 430000 |\n",
+       "+-----+------+-------+-----+---+----------------+---------------+------------------+--------+"
+      ]
+     },
+     "execution_count": 17,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
    ],
+   "source": [
+    "# removes columns \"latitude\" and \"longitude\" from table\n",
+    "pricing_columns = pricing.remove_columns([\"latitude\", \"longitude\"])\n",
+    "# removes rows which contain missing values\n",
+    "pricing_values = pricing_columns.remove_rows_with_missing_values()\n",
+    "# removes rows which contain outliers\n",
+    "pricing_outliers = pricing_values.remove_rows_with_outliers()\n",
```

Comment:
Could also chain the method calls, since we never refer to the intermediate results later.